codecov.yml

```diff
@@ -0,0 +1,8 @@
+comment:                  # this is a top-level key
+  layout: "reach, diff, flags, files"
+  behavior: default
+  require_changes: false  # if true: only post the comment if coverage changes
```

Comment:
Should we set this to true? No strong opinions but kinda feels unnecessary if there is no coverage change

Reply:
Yes, it makes sense. I kept it only to test this.